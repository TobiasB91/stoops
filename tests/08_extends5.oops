CLASS A IS
    METHOD B IS
    BEGIN
        WRITE 65;
    END METHOD
END CLASS

CLASS B IS
END CLASS

CLASS Main EXTENDS A IS
    METHOD b : B IS | B ist Funktion, kein Typ
    BEGIN
        RETURN NULL;
    END METHOD

  METHOD main IS
  BEGIN
      B;
  END METHOD
END CLASS
