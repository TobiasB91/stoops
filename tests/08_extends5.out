Fehler in Zeile 12, Spalte 16: Typ erwartet
